### ISSUE TITLE
unresolved dependency 

### ISSUE THREAD

**Issue by rdingwell** [haffonist/igamt#405]
Trying to build the project results in an unresolved dependency issue for:

		<dependency>
			<groupId>gov.nist.healthcare.mu.core.stat</groupId>
			<artifactId>stat-logging</artifactId>
			<version>0.0.1</version>
		</dependency>

I imagine that this is hosted either somewhere within NIST or is in a specific repository that needs to be included but is not stated.

**Comment by haffonist** [comment]
yes, we are progressively working on removing those internal dependencies. 
Thanks
